src/raster/array.jl

```diff
@@ -21,9 +21,10 @@ us to be able to index into it like we would an array.
 Constructing a RasterDataset will error if the raster bands do not have all the
 same size and a common element data type.
 """
-struct RasterDataset{T,DS<:AbstractDataset} <: AbstractDiskArray{T,3}
+struct RasterDataset{T,DS<:AbstractDataset,C} <: AbstractDiskArray{T,3}
     ds::DS
     size::Tuple{Int,Int,Int}
+    chunks::C
```

Comment:
Might it be worth giving some kind of indication (maybe as a comment) what the chunk type `C` is like?

Reply:
https://github.com/yeesian/ArchGDAL.jl/pull/462/commits/e1b8d101059574b2db900ccbde9ce497d1c4dcaf